### ISSUE TITLE
(Recommendation of improvements) Backups

### ISSUE THREAD

**Issue by MakuSensei** [inorichi/tachiyomi#2249]
Hello, I would like to recommend some functions to make backup copies better.
Before starting to make the backup, scan the installed extensions and create a txt listing those extensions. With that it would be easier to restore avoiding the typical source import error you will not find. It is also an advantage for those who have many extensions installed and we forget to install some, or for newbies who do not know that they must install the extensions before restoring.
Gracias

**Comment by arkon** [comment]
Marking as duplicate of #1810
